api-pipeline.yml

```diff
@@ -20,21 +20,21 @@ trigger:
 strategy:
   matrix:
     linux:
-      imageName: 'ubuntu-16.04'
+      imageName: 'ubuntu-18.04'
     mac:
-      imageName: 'macos-10.13'
+      imageName: 'macOS-10.14'
     windows:
-      imageName: 'vs2017-win2016'
+      imageName: 'windows-2019'
 
 pool:
   vmImage: $(imageName)
 
 steps:
 
 - task: DotNetCoreInstaller@0
-  displayName: 'Use .NET Core sdk 2.2.107'
+  displayName: 'Use .NET Core sdk 3.0.100'
```

Comment:
During the PR for FE unit tests, I created a folder `azure-devops`. I didn't move these `yml` files because it was out of scope.. but maybe we can move them now as part of this PR?

Reply:
oh I didn't realize there were two files. My bad.
Right, I'll move them and get them to pass. Tks